kit/azure/organization-hierarchy/modules/organization_hierarchy/main.tf

```diff
@@ -0,0 +1,50 @@
+resource "azurerm_management_group" "parent" {
```

Comment:
having organization_hierarchy as a nested submodule feels a bit like one level of nesting too much, can we pull this up one level? I think I prefer splitting up into multiple `resources.*.tf` files instead and have less passing around of vars and documentation output